pkg/crinit/options/options.go

```diff
@@ -103,6 +110,13 @@ func (o *SubcommandOptions) BindCommon(flags *pflag.FlagSet, defaultServerImage,
 		"Preferred port to use for the cluster registry API server NodePort service. Set to 0 to randomly assign a port. Valid only if '"+APIServerServiceTypeFlag+"=NodePort'.")
 }
 
+// BindCommonDelete adds the common options that are shared by the aggregated and
+// standalone delete sub-commands to the list of flags.
+func (o *SubcommandOptions) BindCommonDelete(flags *pflag.FlagSet) {
+	flags.BoolVar(&o.IgnoreErrors, "ignore-errors", false,
+		"Run the command and ignore any errors encountered.")
```

Comment:
Maybe "Ignore errors while deleting resources. This will continue to delete all resources created by `init` even if an error occurs while deleting one of these resources.

Reply:
I've updated the usage to a more concise version of your suggestion.